service/aesEncoder.go

```diff
@@ -0,0 +1,61 @@
+package service
+
+import (
+	"awesomeProject3/service/de_encoding"
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/hex"
+	"io"
+)
+
+type aesEncoder struct {
+	key []byte
+}
+
+func (en aesEncoder) Decrypt(ct string) string {
+	ciphertext, _ := hex.DecodeString(ct)
+
+	c, err := aes.NewCipher(en.key)
+	checkError(err)
+	gcm, err := cipher.NewGCM(c)
+	if err != nil {
+		panic(err)
+	}
+
+	nonceSize := gcm.NonceSize()
+	if len(ciphertext) < nonceSize {
+		panic(err)
```

Comment:
That's not when you should do panic. Is that really exceptional situation that you cannot restore from and handle it properly? Return error instead of panic, for the sake of simplicity and clarity please do not use `panic` in this project at all.